### PR TITLE
Issue 171 - fixed properly closing files upon untarring.

### PR DESCRIPTION
**Pull request by soltysh**
This is to address problem from #171.
@mfojtik @bparees PTAL

//fyi @Maxim-Filimonov

### CODE REVIEW

File: pkg/tar/tar.go [hunk 1]
```diff
@@ -232,3 +196,31 @@ func (t *stiTar) ExtractTarStream(dir string, reader io.Reader) error {
 		}
 	}
 }
+
+func createFile(dir string, header *tar.Header, tarReader io.Reader) error {
```
Comment (mfojtik) [inline pkg/tar/tar.go]: this is not really creating file but extracting file ;-) "extractTARFile"?

Comment (soltysh) [inline pkg/tar/tar.go]: I'll go with just `extractFile` :wink:

**Comment by mfojtik** [comment]
LGTM [merge]

**Comment by Maxim-Filimonov** [comment]
@soltysh thanks a lot :+1: will bot create release automatically?

**Comment by mfojtik** [comment]
@Maxim-Filimonov we are working on release just now, should happen later today (hopefully)

**Pull request merged**